lib/runner.js

```diff
@@ -139,7 +139,7 @@ async function initializeApp(options, elmModulePath, reviewElmJson, appHash) {
      * @returns {never}
      */
     (errors) => {
-      Report.report(options, {errors});
+      void Report.report(options, {errors});
```

Comment:
I'm not sure I knew that `worker.terminate()` returned a `Promise` back when I wrote it, but I think all the floating promises in this commit make sense to keep as floating (for concurrency purposes, etc.).

The only one I'd be worried about it potentially this one, but so far it has always worked well so... I think it's fine to keep as is :shrug: